### ISSUE TITLE
Don't show quiet network results in fast network popups (and vice versa)

### ISSUE THREAD

**Issue by Robinlovelace** [nptscot/nptscot.github.io#149]
As discussed with @mvl22 and Angus. I think this is clear but let me know if you'd like to clarify anything.

**Comment by Robinlovelace** [comment]
@mvl22 should we make this a Phase 3 issue? Sense it may be a lot of work for you.

**Comment by mvl22** [comment]
> should we make this a Phase 3 issue? Sense it may be a lot of work for you.

Yes, please. There's not a clear way to do this yet I've found, but I hope future refactoring will make it so.

**Comment by Robinlovelace** [comment]
My view: we should have different `.pmtiles` files for each network type.

**Comment by Robinlovelace** [comment]
That will also reduce the number of columns in the data by ~3x, good from frontend code perspective too I imagine.

**Comment by mvl22** [comment]
Could I get clarification on this issue, in the form of a screenshot demonstrating the problem? I must admit I've slightly lost track of the actual problem here.

**Comment by Robinlovelace** [comment]
The 2 tables in the All network information popup is a bit overwhelming, right? 

![image](https://github.com/user-attachments/assets/4471c71f-da01-4d88-ac91-46aed5201cfb)

**Comment by mvl22** [comment]
That's clearly an old screenshot as the accordion is visible. It should now be scrolling.

**Comment by Robinlovelace** [comment]
OK closing for now. There are still issues, we can open a new issue if there are issues (I suspect there are but this issue is now a bit stale).
